var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -47,14 +48,18 @@ class Cp2k(Package):
     depends_on('lapack')
     depends_on('blas')
     depends_on('fftw')
-    depends_on('libint@:1.2', when='@3.0')
+    depends_on('libint@:1.2', when='@3.0,4.1')
 
     depends_on('mpi', when='+mpi')
     depends_on('scalapack', when='+mpi')
     depends_on('plumed+shared+mpi', when='+plumed+mpi')
     depends_on('plumed+shared~mpi', when='+plumed~mpi')
-    depends_on('pexsi', when='+mpi')
-    depends_on('wannier90', when='+mpi')
+    depends_on('pexsi+fortran', when='+mpi')
+
+    # Apparently cp2k@4.1 needs an "experimental" version of libwannier.a
+    # which is only available contacting the developer directly. See INSTALL
+    # in the stage of cp2k@4.1
```

Comment:
Ah, I see. Once the next version of `wannier90` is released we can go back and re-add support then. I'm surprised `cp2k` would release a version that doesn't support the latest version of `wannier90`.